packages/ai-jsx/src/experimental/next/index.ts

```diff
@@ -145,3 +147,8 @@ export const jsx = asJsxBoundary(function jsx(
   }) as JSX.Element;
 });
 export const JSX = jsx;
+
+export function toReactStream(componentMap: ComponentMap<any>, renderable: AI.Renderable): Response {
+  const renderResult = AI.createRenderContext().render(renderable, { stop: (e) => e.tag == AI.React, map: (x) => x });
```

Comment:
minor: is there a reason not to make the identity function the default value for `map`?

Reply:
Sadly yes -- it's a speed-bump to make it harder to accidentally do the wrong thing with partial rendering. (I'm inclined to condemn partial rendering to a lower-level API in the future because it's IMO not worth the ambiguity around attempting to make it ergonomic.)